Firestore/Example/Tests/API/CMakeLists.txt

```diff
@@ -12,27 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if(APPLE)
-  firebase_ios_objc_test(
-    firebase_firestore_objc_api_test
-    HOST firebase_firestore_example_app
-    SOURCES
-      FIRCollectionReferenceTests.mm
-      FIRDocumentReferenceTests.mm
-      FIRDocumentSnapshotTests.mm
-      FIRFieldPathTests.mm
-      FIRFieldValueTests.mm
-      FIRFirestoreTests.mm
-      FIRGeoPointTests.mm
-      FIRQuerySnapshotTests.mm
-      FIRQueryUnitTests.mm
-      FIRSnapshotMetadataTests.mm
-      FIRTimestampTest.m
-      FSTAPIHelpers.h
-      FSTAPIHelpers.mm
-      FSTUserDataConverterTests.mm
-    DEPENDS
-      FirebaseFirestore
-      firebase_firestore_objc_test_util
-  )
-endif(APPLE)
+if(NOT FIREBASE_IOS_BUILD_TESTS OR NOT APPLE)
```

Comment:
Testing the value of `FIREBASE_IOS_BUILD_TESTS` and `APPLE` is different from the original file, which only tested `APPLE`.  Can you confirm that this is an intended change?